blueprints/foundational/config.ts

```diff
@@ -0,0 +1,293 @@
+import {
+  baseConfig,
+  Config,
+  GLOBAL_REGION,
+  Template,
+} from '@superluminar-io/aws-luminarlz-cli/lib/config';
+
+/**
+ * The deployed version of the AWS Landing zone accelerator.
+ */
+export const AWS_ACCELERATOR_VERSION = '1.12.1';
+
+/**
+ * The AWS management account, organization and root organizational unit used in this project.
+ */
+export const MANAGEMENT_ACCOUNT_ID = '<<AWS_MANAGEMENT_ACCOUNT_ID>>';
+export const ORGANIZATION_ID = '<<AWS_ORGANIZATION_ID>>';
+export const ROOT_OU_ID = '<<AWS_ROOT_OU_ID>>';
+
+/**
+ * The email address used for the AWS account root emails.
```

Comment:
for which account is this actually? the root account, right?

```suggestion
 * The email address used for the AWS root account root emails.
 * TODO: update with the root email used in the account.
```

Reply:
This wouldn't be correct since the blueprint assumes that one email group with sub-addressing is used for all root email addresses.

Reply:
I'll add another commit which includes the TODO.

Reply:
do you mean "root email address" as opposed to sub addresses? There are too many "roots" this can refer to. sorry for re-opening, i guess there is a misunderstanding about "root" and we need to clarify that.